### PR TITLE
Enable sparse support for SVM's for oneAPI interfaces

### PR DESCRIPTION
**Pull request by agorshk**

### CODE REVIEW

File: cpp/oneapi/dal/algo/svm/detail/kernel_function.cpp [hunk 1]
```diff
@@ -106,25 +117,29 @@ class daal_interop_rbf_kernel_impl : public kernel_function_impl {
...
-        return daal_kf_t(alg);
+    daal_kf_t get_daal_kernel_function(bool is_dense) override {
+        if (is_dense) {
+            constexpr daal_rbf_kernel::Method daal_method = get_daal_method(true);
+            auto alg = new daal_rbf_kernel::Batch<Float, daal_method>;
+            alg->parameter.sigma = sigma_;
+            return daal_kf_t(alg);
+        }
+        else {
+            constexpr daal_rbf_kernel::Method daal_method = get_daal_method(false);
```
Comment (PetrovKP) [inline cpp/oneapi/dal/algo/svm/detail/kernel_function.cpp:128]: better separate method: `get_daal_dense_method` and `get_daal_csr_method`

File: cpp/oneapi/dal/table/detail/csr.hpp [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+    /// performed that this type is the actual type of the data within the table.
+    template <typename Data>
+    const Data* get_data() const {
+        return reinterpret_cast<const Data*>(this->get_data());
+    }
+
+    /// The pointer to the data block within the table.
+    /// Should be equal to ``nullptr`` when :expr:`row_count == 0` and :expr:`column_count == 0`.
+    const void* get_data() const;
+
```
Comment (PetrovKP) [inline cpp/oneapi/dal/table/detail/csr.hpp:99]: need method: get_non_zero_count

File: cpp/oneapi/dal/backend/interop/table_conversion.hpp [hunk 3]
```diff
@@ -122,12 +123,147 @@ inline daal::data_management::NumericTablePtr convert_to_daal_table(const homoge
     return copy_to_daal_homogen_table<Data>(table);
 }
 
+template <typename T>
+inline auto convert_to_daal_csr_table(array<T>& data,
```
Comment (PetrovKP) [inline cpp/oneapi/dal/backend/interop/table_conversion.hpp:127]: merge with homogen as example: 
convert_to_daal_table -> convert_to_daal_homogen_table, convert_to_daal_csr_table

convert_from_daal_table -> convert_from_daal_homogen_table, convert_from_daal_csr_table

**Comment by PetrovKP** [comment]
/intelci: run
set daal4py_branch=rls/2021.2.3-rls

**Comment by PetrovKP** [comment]
/intelci: restart
set daal4py_branch=csr-support-onedal

File: cpp/oneapi/dal/backend/interop/table_conversion.hpp (hunk 3 above)
Comment (agorshk) [reply cpp/oneapi/dal/backend/interop/table_conversion.hpp]: Ok, it was fixed

File: cpp/oneapi/dal/table/detail/csr.hpp (hunk 2 above)
Comment (agorshk) [reply cpp/oneapi/dal/table/detail/csr.hpp]: Ok, it was added

File: cpp/oneapi/dal/algo/svm/detail/kernel_function.cpp (hunk 1 above)
Comment (agorshk) [reply cpp/oneapi/dal/algo/svm/detail/kernel_function.cpp]: Ok, it was fixed

**Review by PetrovKP — APPROVED** [review]
Merge after green CI

**Pull request merged**